### ISSUE TITLE
Documentation needs more explanation.

### ISSUE THREAD

**Issue by Pranjal-Srivastava-2023** [ubsuny/23-Homework4G2#79]
@JustinJaniszewski the structure of the documentation looks good but we need to explain it in more detail. For example, what different command in .yml files do. We also modified the Unittesting.yml according to our code and incorporated matplotlib and other libraries but these changes are not mentioned in the documentation.

**Comment by JustinJaniszewski** [comment]
Understood, will do that

**Comment by Pranjal-Srivastava-2023** [comment]
@JustinJaniszewski Please close the issue when the task is completed. That's the point of creating milestones so that we can track the progress.

**Comment by JustinJaniszewski** [comment]
Okay

**Comment by s4il3sh** [comment]
Agreed! I hope @JustinJaniszewski is working on it to provide a complete picture. Let me know if anything has to be done. I could lend you a hand if you want to.

**Comment by AhmedCode99** [comment]
good
